config/apt_constants.go

```diff
@@ -24,7 +24,7 @@ const (
 	// currently configured APT source location. We assume that
 	// the first source for "main" in the file is the one that
 	// should be replaced throughout the file.
-	ExtractAptSource = `awk "/^deb .* $(lsb_release -sc) .*main.*\$/{print \$2;exit}" ` + AptSourcesFile
```

Comment:
`lsb_release` command is not always available, eg minimal Ubuntu image used for k8s jujud agent